cmd/global-query/pkg/distributed/query.go

```diff
@@ -69,19 +58,11 @@ func (q *QueryRunner) Run(ctx context.Context, args *query.Args) (*results.Resul
 	// log the query
 	logger := logging.Logger().With("hosts", hostList)
 
-	// query pipeline setup
-	// sets up a fan-out, fan-in query processing pipeline
-	numRunners := len(hostList)
-	if q.maxConcurrent > 0 && q.maxConcurrent < numRunners {
-		numRunners = q.maxConcurrent
-	}
+	logger.Info("reading query results from querier")
```

Comment:
Do we need / want that as `Info` level? Either it succeeds, then we get results / info someplace else, or it fails, in which case we get error(s)...

Reply:
I do want to use it, yes. Will adjust in case it gets too noisy.